src/app/views/research-new/research-new.component.html

```diff
@@ -7,30 +7,140 @@
 
           <!-- -----------Section - 2 --------------------- -->
 
-<h3 class="bold p-5 pt-4" style="font-size: 50px;">Research</h3>
-
-<div class="p-5 row w-100 bg-light" >
-  <div class="col-12 shadow col-md-3  align-middle align-items-center card p-2 py-5" style="height:fit-content">
-    <h2 class="my-2 mb-5">Contents</h2>
-    <ul style="  font-size: 25px;">
-      <li>About</li>
-      <li>Projects</li>
-      <li>Vignettes/Gallery</li>
-      <li>Collaborations </li>
-      <li>Facilities & Labs</li>
-    </ul>
+<div class="p-5 row w-100 bg-light">
+<div class="col-12 shadow px-5 p-2">
+  <h1 class="my-2 mb-5"><b>ReSearch : </b> Contents</h1>
```

Comment:
S should not be in capital in word Research

Reply:
Yeah Okay. I put it maybe to look creative but tha's fine.